model/src/main/kotlin/OrtResult.kt

```diff
@@ -54,13 +54,7 @@ data class OrtResult(
      * An [EvaluatorRun] containing details about the evaluation that was run using the result from [scanner] as
      * input. Can be null if no evaluation was run.
      */
-    val evaluator: EvaluatorRun? = null,
-
-    /**
-     * A map that holds arbitrary data. Can be used by third-party tools to add custom data to the model.
-     */
-    @JsonInclude(JsonInclude.Include.NON_EMPTY)
-    val data: CustomData = emptyMap()
+    val evaluator: EvaluatorRun? = null
```

Comment:
Commit message: "auto-generated functions", as in previous commit.

Reply:
Done.

Reply:
Typo in "functions".

Reply:
Done.